cloudvcl/settings.py

```diff
@@ -119,6 +119,10 @@
 
 STATIC_URL = '/static/'
 
+# Drexel Domain
```

Comment:
So you effectively tell everyone to put "drexel.edu" below? How is that making it configurable?
You must provide a clear description of what should be typically entered and how it is going to be used.